src/__tests__/Error.test.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import Error from '../components/Error';
+
+describe('Error component', () => {
+  it('should render an error message', () => {
+    const wrapper = shallow(<Error />);
+    expect(
+      wrapper.contains(
+        <p>Error: Invalid input. Please input a number from 3 - 10</p>
```

Comment:
There's nothing that _needs_ to change here, but on the subject of reusability: how reusable is this `Error` component? Could I use it on a login form? If not, does the name indicate that I shouldn't use it on a login form (or where it _should_ be used)?